### PR TITLE
fix(storybook): add keys to theme components

### PR DESCRIPTION
**Pull request by jendowns**
Right now in development, when viewing the storybook environment, you will see a large warning about a miss `key` value on the top level `Theme`:

Screen Shot 2020-05-20 at 12 51 15 PM

## Proposed changes

- the component wrapper `Theme` will have a `key={NAME-OF-THEME}` (this is not the specific missing `key` being flagged but still seems useful, anyway)
- the `Theme` addon panel will have a `key` that matches naming convention of other addon panels -- e.g., `key="storybook/theme/panel"`

### CODE REVIEW

File: .storybook/addons/addon-theme/index.js [hunk 1]
```diff
@@ -61,7 +61,11 @@ class Wrapper extends Component {
     this.setState({ theme });
   };
 
-  render = () => <Theme theme={this.state.theme}>{this.props.children}</Theme>;
+  render = () => (
```
Comment (SimonFinney) [inline .storybook/addons/addon-theme/index.js:64]: 👏

File: .storybook/addons/addon-theme/index.js [hunk 2]
```diff
@@ -61,7 +61,11 @@ class Wrapper extends Component {
...
+    <Theme key={this.state.theme} theme={this.state.theme}>
```
Comment (SimonFinney) [inline .storybook/addons/addon-theme/index.js:65]: 👏

File: .storybook/addons/addon-theme/register.js [hunk 3]
```diff
@@ -58,7 +58,12 @@ register(namespace, api =>
   addPanel(`${namespace}/panel`, {
     title,
     render: ({ active }) => (
-      <Theme active={active} api={api} channel={getChannel()} />
+      <Theme
+        key="storybook/theme/panel"
```
Comment (SimonFinney) [inline .storybook/addons/addon-theme/register.js:62]: Nice! I wasn't familiar with this convention, where can I find more information about it? Perhaps in a follow-up, we update the `namespace`

Comment (jendowns) [reply .storybook/addons/addon-theme/register.js:62]: You can see the keys of other storybook panels if you inspect the storybook environment using React DevTools:

Screen Shot 2020-05-21 at 8 20 11 AM

I don't recommend trying to edit those keys though.

**Pull request merged**